test/test_decomp.py

```diff
@@ -412,6 +427,8 @@ def _torch_dispatch(self, func, types, args=(), kwargs=None):
 
                 # Stuff we shouldn't bother testing
                 # (TODO: remove detach from the decomp table?)
+                # N.b. Testing in-place ops would need dedicated logic
+                in_place = func.name()[-1] == '_'
```

Comment:
@anjali411 don't we have a more direct way of querying if an OpOverload is inplace now?